test/rubber_tree.md

```diff
@@ -0,0 +1,9 @@
+# Rubber Tree
+
+1. Run client.
+2. Create a new world (creative world).
+3. Fly around to find a rubber tree.
```

Comment:
Should we add some more remarkable picture for our featured creatures? And add some build option to control this.

I think that will help debugging/testing.

Reply:
> Should we add some more remarkable picture for our featured creatures? And add some build option to control this.
> 
> I think that will help debugging/testing.

"build option" in JVM……Might be hard.